module-3/app/service/client/mysfitsTableClient.go

```diff
@@ -0,0 +1,188 @@
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+// Info is for logging
+var Info *log.Logger
+
+// Init initializes the logger
+func Init(
```

Comment:
would like to see the use of the standard golang init method here with no arguments and a sensible default, or to make this entire set of functions in this file a struct with a constructor that allows for an io.Writer argument